.gitignore

```diff
@@ -35,5 +35,5 @@ lib
 es
 node_modules
 npm-debug.log*
-yarn.lock
+lerna-debug.log*
```

Comment:
Should also gitignore + git remove `package-lock.json` (It's also got a security warning flagged in GH web interface)